### ISSUE TITLE
Fallout 4 Folder Error

### ISSUE THREAD

**Issue by BrendanoElTaco** [Fallout-London/FOLON-FO4Downgrader#8]
System: Windows 11 Pro

Downgrader Version: v1.5

Issue: Whenever the folder icon is clicked to open a file directory to select the fallout 4 install location an error pops up. The error reads:
```
Traceback (most recent call last):
  File "FOLON-Downgrader.py", line 259, in GetDirectory
IndexError: list index out of range
```

**Comment by coffandro** [comment]
Don't use the button for now

**Comment by coffandro** [comment]
The error is caused by me trying to be smort, ill need to revert a file change in utility.py to just use directories and now winreg for steam info

**Comment by coffandro** [comment]
fixed in 2ecc81e, will be in version 1.5.2
